plugins/mcreator-core/procedures/explode.json

```diff
@@ -5,6 +5,11 @@
       "name": "power",
       "check": "Number"
     },
+    {
+      "type": "input_value",
+      "name": "condition",
```

Comment:
Make this field more explanatory, "condition" can be a lot of things

Reply:
maybe "fire_condition"?

Reply:
We usually use camel case afaik, preferrably use field name from the method you call